org/epifany/pkmnbreedprbltycalc/PkmnCalcManager.java

```diff
@@ -1,12 +1,14 @@
-package pokemoniv_4;
+/**
+ * Copyright 2016, Stephen Gung, All rights reserved
+ */
+
+package org.epifany.pkmnbreedprbltycalc;
```

Comment:
Not a huge fan of this mashup `pkmnbreedprbltycalc` should be something that's an actual word.